### PR TITLE
[IMP] Sign documentation updated

### PR DESCRIPTION
**Pull request by Donapi**
[ID:3433578](https://www.odoo.com/web#id=3433578&menu_id=4720&cids=1&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/finance/sign.rst [hunk 1]
```diff
@@ -213,20 +234,31 @@ the field before it is completed.
...
+
+By going to :menuselection:`Configuration --> Settings`, you can :guilabel:`Enable Signing Order` to
+allow users to define a signing order, where recipients are only notified when it is their turn to
+sign.
+
+Expiration and reminders
+------------------------
+
+It is possible to set a **validity date** to your document or to send **automatic email reminders**
+to people who still need to sign a document after a set delay.
```
Comment (zulu-odoo) [inline content/applications/finance/sign.rst:260]: It is possible to set a **validity date** to your document. To obtain the signature on time, you can set up **Automatic email reminders** for people who still need to sign a document.

**Review by zulu-odoo — APPROVED** [review]
Hi Dona,

As discussed, feel free to modify the doc after our comments :)

Thanks

File: content/applications/finance/sign.rst [hunk 2]
```diff
@@ -213,20 +234,45 @@ the field before it is completed.
...
-example) to indicate that the signature is electronic and that there is some traceability behind
-it. The :guilabel:`hash` corresponds to a unique ID number related to the signed document.
+Tags can be used to categorize and organize documents, and allow users to easily search for and
+filter documents based on specific criteria.
 
-If you are an :doc:`internal user <../general/users/manage_users>`,
-you can choose to tick or untick the frame option when signing the document. Tick the box to have
-the frame and hash visible.
+You can manage tags by going to :menuselection:`Configuration --> Tags`. To create a tag, click
+:guilabel:`New`. A line is created. Add the :guilabel:`tag name` and select a
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/finance/sign.rst:244]: let's discuss this IRL

File: content/applications/finance/sign.rst [hunk 3]
```diff
@@ -101,71 +139,124 @@ template.
...
+Upon signing the document, the signer completes the :guilabel:`Signature` field and proceeds by
+clicking on :guilabel:`Validate & Send Completed Document`, triggering a
+:guilabel:`Final verification` page where authentication via itsme® is required.
 
-.. note::
-   This feature is only available in Belgium and the Netherlands.
+Signatory hash
+==============
+
+Each time someone signs a document, a hash - a unique ID number - is generated to ensure
```
Comment (bouvyd) [inline content/applications/finance/sign.rst:206]:
```suggestion
Each time someone signs a document, a hash - a unique digital signature of the operation - is generated to ensure
```

Comment (bouvyd) [reply content/applications/finance/sign.rst:206]: i want to make it somewhat clear that the _content_ the person used for signing (their name, their signature, the input fields they filled in) is used to compute this signature - so any alteration will corrupt the whole 'hash chain' and be visible to the system

here's how chatgpt puts it, which is not a bad start:
> In Odoo Sign, the integrity and inalterability of documents are safeguarded using a hashing technique. At each stage of the signing process, a unique hash is created based on the document's current content and the cumulative hashes of previous steps. This includes any information added by users, like names and signatures. This method ensures that any changes made after a signature is affixed can be easily detected, maintaining the document's authenticity and security throughout its lifecycle."

File: content/applications/finance/sign.rst [hunk 4]
```diff
@@ -101,71 +139,124 @@ template.
...
+
+To apply a tag to a document, use the dropdown list available in your document.
+
+Sign order
+==========
+
+When a document needs to be signed by different parties, the signing order lets you control the
+order in which your recipients receive it for signature.
+
+By going to :menuselection:`Configuration --> Settings`, you can :guilabel:`Enable Signing Order`.
```
Comment (bouvyd) [inline content/applications/finance/sign.rst:236]: no longer needed in 17 btw, signing order is always available as soon as you have more than 1 signer
image

Comment (bouvyd) [reply content/applications/finance/sign.rst:236]: image

Comment (Donapi) [reply content/applications/finance/sign.rst:244]: I keep it in mind for V17, thanks!

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**